src/components/MyData/LineChart/AxisArrows.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { useCommonChartProps } from '../useCommonChartProps';
+import { BlockProps } from 'victory-core';
+import { useTheme } from '../../../hooks';
+import { Trace } from './TraceLine';
+import { useVictoryTheme } from '../useVictoryTheme';
+
+type Props = {
+  trace1: Trace;
+  trace2?: Trace;
+};
+export const AxisArrows = ({ trace1, trace2 }: Props) => {
+  const theme = useTheme();
+  const commonProps = useCommonChartProps();
+  const trace1Theme = useVictoryTheme(trace1, 'trace1');
+  const trace2Theme = useVictoryTheme(trace2, 'trace2');
+
+  const width = commonProps.width ?? 0;
+  const height = commonProps.height ?? 0;
+  const padding: Required<BlockProps> = new Proxy({} as any, {
```

Comment:
Cool! I wasn't even sure what this did exactly without looking it up but it is a very elegant way of defaulting the inner properties of padding.